src/main/resources/Ideas/IdeasTranslations.xml

```diff
@@ -92,6 +92,7 @@ ideas.livetable.priority=Priority
 
 # Ideas application keys
 ideas.home.description=Ideas worth spreading
+ideas.ideasclass.vote.closed=Voting is not available
```

Comment:
This doesn't provide much information about the reason behind.

Reply:
Would have been better to have a notification for when the vote is closed and one for when it is rejected? Or something like 'The idea is closed / rejected'?